.template/addons/github/.github/workflows/deploy_aws_fargate.yml.tt

```diff
@@ -0,0 +1,50 @@
+name: Deploy to AWS Fargate
```

Comment:
I think ECS is a correct name 🤔  we are deploying to the ECS and the ECS will handle the deployment to Fargate 🤔

Reply:
Fargate is a part of ECS but AWS Fargate is still the correct name, according to Amazon 😃
https://aws.amazon.com/fargate/
![Screen Shot 2021-03-24 at 13 47 45](https://user-images.githubusercontent.com/1896814/112267093-885aa100-8ca7-11eb-972a-eac785f3ea2b.png)

Reply:
but we are connecting to the ECS, not the Fargate right, base on the script in this file.

Reply:
In this script, we must use the ECS command to deploy because Fargate is part of ECS. This script is created to target Fargate deployment only, not the EC2 with EC2. By naming `Deploy to AWS Fargate`, it removes the confusion that this workflow supports ECS with EC2 deployment.

Reply:
Please correct me if I'm wrong but I think `aws ecs update-service --cluster $ECS_CLUSTER --service $ECS_SERVICE --force-new-deployment` also work with EC2, there is nothing relative to Fargate or EC2 from this point 🤔.

Reply:
@andyduong1920  it should work but I haven't had a chance to test it against EC2 deployment on ECS :D But I'm sure it works for Fargate since we used it in Nimble CX.

For EC2 deployment, we used different way in The-API https://github.com/nimblehq/the1-api/blob/development/bin/ecs-deploy

Reply:
@andyduong1920 I removed the workflow to deploy to ECS from this PR. It will be handled by #289